coffee-chats/src/main/java/com/google/step/coffee/data/GroupStore.java

```diff
@@ -0,0 +1,37 @@
+package com.google.step.coffee.data;
+
+import com.google.appengine.api.datastore.*;
+import com.google.step.coffee.UserManager;
+import com.google.step.coffee.entity.Group;
+
+public class GroupStore {
+  private DatastoreService datastore;
+
+  public GroupStore() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  public Group create(String name) {
+    return put(Group.builder()
+        .setName(name)
+        .setDescription("")
+        .setOwnerId(UserManager.getCurrentUserId())
+        .build());
+  }
+
+  public void delete(Key key) {
+    datastore.delete(key);
+  }
+
+  public Group put(Group group) {
```

Comment:
Need comments. How is group in argument different from return value?

Reply:
Ideally, every function should have comments. Probably with an exception of functions with length of one line only